pkgs/development/compilers/llvm/6/clang/default.nix

```diff
@@ -51,13 +53,17 @@ let
     outputs = [ "out" "lib" "python" ];
 
     # Clang expects to find LLVMgold in its own prefix
-    # Clang expects to find sanitizer libraries in its own prefix
+    # Clang expects to find sanitizer libraries and headers in its own prefix
     postInstall = ''
```

Comment:
Anyone have details about this?

Also discussed elsewhere in this PR (https://github.com/NixOS/nixpkgs/pull/39743#discussion_r189405952) -- I'd just like to know about the issues, as it'd be awfully nice to not have to link these bits over... (perhaps combined with a wrapper or so).

Reply:
I'll see if it's possible to get away with `-L` and `-isystem` for these. The approach taken here was based on the existing code and the expectations of the clang developers, but it isn't necessarily impossible to do otherwise. If it works out, how would I go about injecting them gracefully?

Reply:
Sounds good! If it's a big deal, we can look into it later--don't mean to push all of our LLVM-related TODO's onto your PR! :grin:

But if you don't mind taking a look that'd be great! Thanks!